test/integration/estimates.test.js

```diff
@@ -46,7 +46,7 @@ describe('Estimates Integration', function () {
     );
     const estimate = createEstimateResponse.data;
 
-    expect(estimate.type).to.be.eq('shipping');
+    expect(estimate.type).to.be.eq('ecommerce');
```

Comment:
We're now re-routing to the ecommerce endpoint: https://github.com/patch-technology/patch/pull/5588